packages/rum/src/resourceUtils.ts

```diff
@@ -57,7 +58,9 @@ function areInOrder(...numbers: number[]) {
   return true
 }
 
-export function computePerformanceResourceDuration(entry: PerformanceResourceTiming): number {
+export function computePerformanceResourceDuration(
+  entry: RumPerformanceResourceTiming | PerformanceResourceTiming
```

Comment:
`RumPerformanceResourceTiming extends PerformanceResourceTiming` would avoid that